standstill_action.talon-list

```diff
@@ -0,0 +1,28 @@
+list: user.clickless_mouse_action
```

Comment:
I generally recommend naming the file the same as the list name for ease of searchability